plugin/percona-pam-for-mysql/src/groups.c

```diff
@@ -17,19 +17,21 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 
 #include <pwd.h>
 #include <grp.h>
+#include <errno.h>
+#include <my_sys.h>
 #include <string.h>
 #include <stdlib.h>
 #include <unistd.h>
 
-enum { max_nss_name_len = 10240 };
-enum { max_number_of_groups = 1024 };
+static int32_t gr_buf_size= 0;
```

Comment:
OK, then at lest gr_buf_size and groups_iter::buf_size types should agree? And it can be plain int, it will be atomic the same